client.ts

```diff
@@ -2,9 +2,9 @@ import { Metadata } from "./types";
 
 const apiURL = "https://contributions.guardianapis.com/epic";
 
-export const getBodyEnd = (meta: Metadata): Promise<Response> => {
+export const getBodyEnd = (meta: Metadata, useUrl?: string): Promise<Response> => {
```

Comment:
There's a '[rule](https://reactjs.org/docs/hooks-custom.html)' in react that custom hooks should always start with `use...` - so, arguably, we maybe shouldn't use the same prefix elsewhere?

Reply:
thanks @oliverlloyd it's true and the param has been renamed. I just wanted to make it clear the argument was meant to be a testing thing/override